Added rust translation of nm2rgb.py

afunsten opened the pull request:
sample target tested on macOS

georgalis reviewed (commented):
Hey @afunsten I'd like to merge this but need a couple changes... please

- purge the binaries and/or tmp files, `tint/Cargo.lock tint/target/release/tint tint/target/.rustc_info.json`
- don't squash the former python, extend to run each lang for comparison, eg demo python, rust, swift for users to choose `tint/fn.bash tint/README.md tint/fn.bash`
- change ./src/ to ./src/rust/ 
- provide a build command and install the binary to eg `./bin/rust/tint` (assume RUSTUP_HOME and CARGO_HOME are in env)

I'll follow these conventions for `nm2rgb.py` after the pr. Would be great to demo, compare/contrast the python/rust/swift versions without collisions!!

afunsten commented:
I reopened the PR and addressed some of the issues.  Please have another look.
_Adam

> On Jul 25, 2022, at 4:04 PM, George Georgalis ***@***.***> wrote:
> 
> 
> @georgalis commented on this pull request.
> 
> Hey @afunsten <https://github.com/afunsten> I'd like to merge this but need a couple changes... please
> 
> purge the binaries and/or tmp files, tint/Cargo.lock tint/target/release/tint tint/target/.rustc_info.json
> don't squash the former python, extend to run each lang for comparison, eg demo python, rust, swift for users to choose tint/fn.bash tint/README.md tint/fn.bash
> change ./src/ to ./src/rust/
> provide a build command and install the binary to eg ./bin/rust/tint (assume RUSTUP_HOME and CARGO_HOME are in env)
> I'll follow these conventions for nm2rgb.py after the pr. Would be great to demo, compare/contrast the python/rust/swift versions without collisions!!
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/georgalis/colab-61cc/pull/1#pullrequestreview-1050177883>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AB2LUITFTR5W3KQX6J24FJ3VV4MRRANCNFSM52U4NBZA>.
> You are receiving this because you were mentioned.
>

afunsten commented:
> Hey @afunsten I'd like to merge this but need a couple changes... please
> 
> * purge the binaries and/or tmp files, `tint/Cargo.lock tint/target/release/tint tint/target/.rustc_info.json`
> * don't squash the former python, extend to run each lang for comparison, eg demo python, rust, swift for users to choose `tint/fn.bash tint/README.md tint/fn.bash`
> * change ./src/ to ./src/rust/
> * provide a build command and command to install the binary to eg `./bin/rust/tint` (assume RUSTUP_HOME and CARGO_HOME are in env)
> 
> I'll follow these conventions for `nm2rgb.py` after the pr. Would be great to demo, compare/contrast the python/rust/swift versions without collisions!!

I reopened the PR and addressed some of the issues.  Please have another look.
